src/app/static/src/app/components/ResetConfirmation.vue

```diff
@@ -1,10 +1,10 @@
 <template>
     <div>
         <modal :open="open">
-            <h4 v-if="isGuest" v-translate="'haveYouSaved'"></h4>
-            <h4 v-if="!isGuest" v-translate="'saveVersion'"></h4>
+                <h4 v-if="isGuest" v-translate="'haveYouSaved'"></h4>
+                <h4 v-if="!isGuest" v-translate="'saveVersion'"></h4>
 
-            <p v-translate="'discardWarning'"></p>
+                <p v-translate="'discardWarning'"></p>
```

Comment:
Hardly a big issue, but this change seems to make the slightly haphazard formatting of this file less consistent, rather than more. Also doesn't seem relevant to rest of PR?

Reply:
This was an accident, thanks for spotting!